### ISSUE TITLE
Migrate to JSR

### ISSUE THREAD

**Issue by RuiNtD** [colinhacks/zod#3300]
Hello. It would be nice if this package could be [migrated to JSR](https://jsr.io/docs/migrate-x-to-jsr), Deno's new preferred registry.

**Comment by colinhacks** [comment]
Thanks, I'll start looking into this.

**Comment by SpeedySH** [comment]
Any progress?

**Comment by dmint789** [comment]
Hono has also migrated its package from deno.land to JSR. Hono even has a zod validator on JSR, so it would be nice to have the base package on there too.

**Comment by halvardssm** [comment]
I’m happy to provide a pr? @colinhacks

**Comment by dmint789** [comment]
> I’m happy to provide a pr? @colinhacks 

There is already a PR linked above, it just hasn't seen any activity lately

**Comment by aurelien-altarriba** [comment]
Any news for Zod on JSR ?

**Comment by chillbrodev** [comment]
@colinhacks Bumping this to see if we can get Zoe available on JSR.

**Comment by 270556660** [comment]
[https://jsr.io/@zod/zod](https://jsr.io/@zod/zod)
The link already exists, but it cannot be found through the search on jsr.

**Comment by chillbrodev** [comment]
> https://jsr.io/@zod/zod The link already exists, but it cannot be found through the search on jsr.

Oh how interesting, thanks for sharing this.

**Comment by mintydev789** [comment]
Is this done now?

**Comment by colinhacks** [comment]
Indeed, `@zod/zod` has been published to JSR since the release of Zod 4

**Comment by BlackAsLight** [comment]
> [https://jsr.io/@zod/zod](https://jsr.io/@zod/zod)
> The link already exists, but it cannot be found through the search on jsr.

The problem with the search functionality is believed to be fixed now and you can now find zod via it
